### DISCUSSION TITLE
Thanks to group

### DISCUSSION THREAD

**Discussion by donarturo11** [muse-sequencer/muse#1124] [Team Posts]
Hi everyone. Thank you for accept me to group. This is a big responsibility and I hope that I won't let anyone down.

**Comment by spamatica** [comment]
Hey there, seem to have been very quiet here. Don't know if you already moved on... if not you are welcome!
You will find that things sadly does not move forward that fast. But you are free to help improve that :)

There aren't that many of us and my coming and going is very intermittent (tends to be more during winter, which is why I'm here ;) )

I think you had been talking mostly with Tim and he wasn't even in this group (added him now.. I think)

**Comment by terminator356** [comment]
Oops I missed that message. Anyway, welcome, if you are still here.
